code/modules/mapping/mapping_helpers.dm

```diff
@@ -851,3 +851,96 @@ INITIALIZE_IMMEDIATE(/obj/effect/mapping_helpers/no_lava)
 	var/turf/open/floor/floor = get_turf(src)
 	floor.burn_tile()
 	qdel(src)
+
+
+///Causes a room to be trashed and graffitied if the revs won in the last round
+/obj/effect/mapping_helpers/revolution_trash
```

Comment:
I think we should avoid this pattern as much as possible, especially since we potentially have a new map along the way.

Why are you doing it like this instead of looking for areas?